### PR TITLE
Securitrons no longer drop stun batons

### PR DESCRIPTION
**Pull request by Superlagg**
[BALANCE]
## About the PR 

Makes securitrons no longer drop stunbatons. Also alters the securitron crafting recipe to require a rod and five cables instead of a stunbaton.

Beepsky totally drops his baton, though!

Couldnt figure out how to make it drop steel rods, though. At least not yet!

## Why's this needed? 

Gotta work a bit harder to get endgame superweapons than knocking over a very knock-overable robit.

## Changelog

```
(u)Superlagg:
(*)Securitrons no longer drop stun batons. Beepsky isn't any ordinary securitron, though!
```

### CODE REVIEW

File: code/modules/robotics/bot/secbot.dm [hunk 1]
```diff
@@ -1152,16 +1156,29 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
...
+		if (W.amount <= 4)
+			boutput(user, "You need a longer length of cable! A length of five should be enough.")
+		else
+			src.build_step++
+			boutput(user, "You add the wires to the rod, completing the Securitron! Beep boop.")
+			var/obj/machinery/bot/secbot/S = new /obj/machinery/bot/secbot(get_turf(src))
+			S.beacon_freq = src.beacon_freq
+			S.hat = src.hat
+			S.name = src.created_name
+			W.amount -= 5
```
Comment (UrsulaMejor) [inline code/modules/robotics/bot/secbot.dm:1180]:
```suggestion
    var/obj/item/cable_coil/C = W
		if (!C.use(5))
			boutput(user, "You need a longer length of cable! A length of five should be enough.")
		else
			src.build_step++
			boutput(user, "You add the wires to the rod, completing the Securitron! Beep boop.")
			var/obj/machinery/bot/secbot/S = new /obj/machinery/bot/secbot(get_turf(src))
			S.beacon_freq = src.beacon_freq
			S.hat = src.hat
			S.name = src.created_name
```

use the use() proc; improper use of use() is how we get cable coils not deleting themselves despite not having any cable in them

File: code/modules/robotics/bot/secbot.dm [hunk 2]
```diff
@@ -1152,16 +1156,29 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
...
+		
+	else if (istype(W, /obj/item/rods) && src.build_step == 3)
+		if (W.amount < 1)
+			boutput(user, "You need a non-zero amount of rods. How did you even do that?")
+		else
+			src.build_step++
+			boutput(user, "You add a rod to [src]'s robot arm!")
+			src.name = "helmet/signaler/prox sensor/robot arm/rod assembly"
+			src.overlays += image('icons/obj/bots/aibots.dmi', "hs_rod")
+			W.amount -= 1
```
Comment (UrsulaMejor) [inline code/modules/robotics/bot/secbot.dm:1168]: Unfortunately, rods don't have a .use() proc. This means you'll need to manually check if the rod stack is empty and clear it out if it's gone:
```
				if (R.amount < 1)
					user.u_equip(R)
					qdel(R)
```

File: code/modules/robotics/bot/secbot.dm [hunk 3]
```diff
@@ -1152,16 +1156,29 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
 		src.name = "helmet/signaler/prox sensor/robot arm assembly"
 		src.overlays += image('icons/obj/bots/aibots.dmi', "hs_arm")
 		qdel(W)
```
Comment (UrsulaMejor) [inline code/modules/robotics/bot/secbot.dm:1158]:
```suggestion
    user.u_equip(W)
		qdel(W)
```

File: code/modules/robotics/bot/secbot.dm (hunk 1 above)
Comment (Superlagg) [reply code/modules/robotics/bot/secbot.dm]: Didn't know that, thanks for letting me know that function exists!

File: code/modules/robotics/bot/secbot.dm (hunk 3 above)
Comment (Superlagg) [reply code/modules/robotics/bot/secbot.dm:1158]: I dont know what user.u_equip(W) is supposed to do here, but I added it anyway and it seems to work!

File: code/modules/robotics/bot/secbot.dm (hunk 2 above)
Comment (Superlagg) [reply code/modules/robotics/bot/secbot.dm:1168]: My bad, forgot that rods don't always auto-delete themselves if their amount goes below 1. Changed!

**Pull request merged**